### PR TITLE
Turn on new Exploration & Analysis and add dataset information

### PR DESCRIPTION
**Pull request by j08lue**
This adds dataset information summaries to be used in the updated Exploration & Analysis feature and turns on the feature.

- [x] Turned on feature
- [x] Added info for `no2`
- [x] Added info for all the other datasets
- [x] Verified that information is shown as expected

## Information required

1. For each dataset, add `infoDescription` to the front matter (see [docs](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/CONTENT.md#datasets)) - will show up here:
   
   image

2. For each layer, add source, spatial extent, temporal resolution, and unit to an `info` block (see [docs](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/frontmatter/layer.md#info)) - will show up here:
   
   image

## Optional

You can also add a thumbnail for each layer that will be shown instead of the dataset thumbnail. See GHG Center exploration for examples of layer thumbnails made from rendered data instead of stock photo https://earth.gov/ghgcenter/exploration

### CODE REVIEW

**Comment by agurvich** [comment]
Alright y'all, I think I managed to get all the metadata in. I didn't realize that we already had it for each dataset under a Dataset Summary heading. I thought I was going to have to figure all of it out myself. I would've done this a lot sooner if I had been paying any attention. Major apologies it's sat this long.

File: datasets/epa-ch4emission-v2express.data.mdx [hunk 1]
```diff
@@ -17,10 +17,19 @@ taxonomy:
   - name: Product Type
     values:
       - Gridded Inventory
+
+infoDescription: | 
+  ::markdown
+      - **Temporal Extent:** 2012 - 2020
```
Comment (aboydnw) [inline datasets/epa-ch4emission-v2express.data.mdx:23]: I think there is one too many indents here, the list is showing up incorrectly in the browser
image

**Comment by aboydnw** [comment]
Thanks @agurvich. I made one small comment on the EPA dataset, otherwise looks good.

Two things, though:
1. Do we need to bump the version of veda-ui to the latest? We are currently on 5.3.1 and I think we will be releasing 5.4.0 this week. Based on the layer selection, I am guessing this is behind.
2. In the GHG Center Portal, EPA stakeholders did not want their data being analyzed against other datasets, so we forced the UI to only allow other EPA datasets to be selected when there is an EPA dataset selected. Should we enable the same logic here? Maybe it's not a concern for earth.gov because the datasets are more clearly disparate than on GHGC. Personally, I think that user experience we have on GHGC layer selection for EPA datasets is a little wonky so I'd say we only replicate it here if we have to.

**Comment by agurvich** [comment]
1. Yes I agree, let's bump the `veda-ui` version number. More on slack, but there are some node version issues that look like we have to merge some commits in from the upstream `veda-config`. 

2. Let's not enable the same logic here. Our POCs at the other agencies are much more hands-off and haven't brought that up. I will touch base with EIC leadership and put on their radar, thanks for the heads-up!

File: datasets/epa-ch4emission-v2express.data.mdx (hunk 1 above)
Comment (agurvich) [reply datasets/epa-ch4emission-v2express.data.mdx:23]: image

Good catch, I thought I looked at all of these... i think it's fixed now

**Comment by aboydnw** [comment]
Found an issue that we resolved in a previous PR, I thought this fix was included in 5.3.1 but maybe it was 5.4.0?

https://www.loom.com/share/4b370cc6f30a421a8175a985f62c7429?sid=a949790f-b802-4b6e-b273-02486730aeac

**Comment by aboydnw** [comment]
Confirming that was 5.4.0 where we fixed this issue, see my comment [here](https://github.com/NASA-IMPACT/veda-ui/pull/1074#issuecomment-2258666775)

**Comment by sandrahoang686** [comment]
@agurvich @aboydnw I have updated this PR to the latest version of veda-ui `v5.5.0` this could use validation from the both of you to make sure everything looks fine with the E&A page the rest of the changes in this PR. I've validated myself for the titiler update from `v5.5.0` and its acting as it should which is requesting the new endpoint, failing (because titiler not yet in prod just yet) and then falling back to old endpoint
Screenshot 2024-08-06 at 4 22 11 PM

**Comment by agurvich** [comment]
Am I supposed to see a chart? If I hover over the area below the tooltip has the statistics in it but I thought there used to be lines: 

image

**Comment by agurvich** [comment]
It's possible there's only one datapoint between the times I had selected so that's why there wasn't a line. 


image

**Review by aboydnw — APPROVED** [review]
Yeah that's probably what happened @agurvich 
I was able to generate time-series charts and everything seems functioning properly. It is interesting to test out the date picker updates with the data on earth.gov, though. Some of the crazy temporal ranges really pressure test our recent changes cc @dzole0311 @faustoperez

**Comment by faustoperez** [comment]
Thanks @agurvich! 

Regarding the single-point results, there's a related design ticket: https://github.com/NASA-IMPACT/veda-ui/issues/961

**Pull request merged**